### ISSUE TITLE
Presentation exchange XXXXX in done state (must be request-received)

### ISSUE THREAD

**Issue by Nabil-oussa** [hyperledger/aries-cloudagent-python#1444]
Hi everyone ! I'm testing the tuto "How to Issue JSON-LD Credentials using Aca-py", "Credential-Specific Contexts" https://github.com/hyperledger/aries-cloudagent-python/blob/main/demo/AliceWantsAJsonCredential.md. I'm blocked in submit the following to Alice's /request-presentation-2.0/{pres_ex_id}/send-presentation. It return this reponse 400: Presentation exchange dc2b55ee-e5bf-4ce7-b13d-1a967fbeb7f7 in abandoned state (must be request-received). 
Someone can help me
Thanks in advance.

**Comment by Tarunshrma** [comment]
@ianco I am also facing this issue intermittantly. I have automated the complete Alice-Faber flow via Jmeter API and 1/10 times I am getting this issue. I have found the console and it seems Alice (Credetial Holder) is giving below error:

"No matching Indy credential found, could not automatically construct presentation for presentation request XXX because referent XXXX did not produce any credential.." 

As discussed this issue is very intermittant and only occures once in a while.

@Nabil-oussa  did you found any solution to this issue.
